test.js

```diff
@@ -5,7 +5,7 @@ import snappy from './snappy';
 import Promise from 'bluebird';
 
 const inputString = 'beep boop, hello world. OMG OMG OMG';
-const inputBuffer = new Buffer(inputString);
+const inputBuffer = Buffer.from(inputString);
```

Comment:
`standard` yells at us since constructor is deprecated.

https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe